ngraph/core/reference/include/ngraph/runtime/reference/deformable_psroi_pooling.hpp

```diff
@@ -0,0 +1,230 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cfenv>
+#include <cmath>
+#include <string>
+#include <vector>
+
+#include "ngraph/shape.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            namespace details
+            {
+                template <typename T>
+                T clamp(const T& value, const T& lower, const T& upper)
```

Comment:
Maybe can we reuse clamp reference implementation?

Reply:
It seems we can, done:)